.github/workflows/ci.yml

```diff
@@ -811,6 +811,12 @@ jobs:
               --benchmark_files="${EXECUTION_BENCHMARK_RESULTS_DIR}/benchmark-results-c2-standard-16.json" \
               --benchmark_files="${EXECUTION_BENCHMARK_RESULTS_DIR}/benchmark-results-a2-highgpu-1g.json" \
               --compile_stats_files="${COMPILE_STATS_RESULTS}"
+      - name: Trigger DB import
+        continue-on-error: true
+        env:
+          DB_IMPORTER_TYPE: ${{ github.event_name == 'pull_request' && 'presubmit' || 'postsubmit' }}
+        run: |
+          build_tools/benchmarks/trigger_db_import.sh "$DB_IMPORTER_TYPE" "${{github.run_id}}" "${{github.run_attempt}}"
```

Comment:
I think in general we follow the pattern to define env variables from github object and use env variables in the command lines. Can you define `GITHUB_RUN_ID` and `GITHUB_RUN_ATTEMPT` for `github.run_id and github.run_attempt?

Reply:
SG. Done. Thx.